content/en/docs/concepts/architecture/coordinated-leader-election.md

```diff
@@ -0,0 +1,56 @@
+---
+reviewers:
+- jpbetz
+title: Coordinated Leader Election
+content_type: concept
+weight: 200
+---
+
+<!-- overview -->
+
+{{< feature-state feature_gate_name="CoordinatedLeaderElection" >}}
+
+Kubernetes {{< skew currentVersion >}} includes an alpha feature that allows
+components to deterministically select a leader via Coordinated Leader Election.
+This is useful to satisfy Kubernetes version skew constraints during cluster upgrades.
+Currently, the only builtin selection strategy is `OldestEmulationVersion`,
+preferring the leader with the lowest emulation version, followed by binary
+version, followed by creation timestamp.
+
+## Enabling Coordinated Leader Election
+
+Ensure that `CoordinatedLeaderElection` [feature
+gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled
+when you start the {{< glossary_tooltip text="API Server"
+term_id="kube-apiserver" >}}: and that the `coordination.k8s.io/v1alpha1` API is
+enabled.
+
+This can be done by setting flags `--feature-gates="CoordinatedLeaderElection=true"` and
+`--runtime-config="coordination.k8s.io/v1alpha1=true"`.
+
+## Component Configuration
+
+With Coordinated Leader Election, components need to both run a LeaseCandidate
+and Lease goroutine (both found in client-go/pkg/leaderelection). Two components
+(kube-controller-manager and kube-scheduler) will automatically use coordinated
+leader election if enabled. Please refer to the example found in
+[k8s.io/kubernetes/cmd/kube-scheduler/app/server.go](https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-scheduler/app/server.go) on set up.
+
+The created LeaseCandidate object looks similar to below:
+
+```
+apiVersion: coordination.k8s.io/v1alpha1
+kind: LeaseCandidate
+metadata:
+  name: hostname_uuid
+  namespace: kube-system
+spec:
+  binaryVersion: 1.31.0
+  emulationVersion: 1.31.0
+  leaseName: kube-scheduler
+  preferredStrategies:
+  - OldestEmulationVersion
+  renewTime: "2024-07-30T03:45:18.325483Z"
+```
+
+Please refer to the [documentation](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#leasecandidate-v1alpha1-coordination-k8s-io) for LeaseCandidate for the full API details.
```

Comment:
If I'm running a cluster, do I need to read that source code? If not, try:
```suggestion
Provided that you have enabled the `CoordinatedLeaderElection` feature gate _and_ have
the `coordination.k8s.io/v1alpha1` API group enabled, control plane components automatically
use the {{< api-reference page="cluster-resources/lease-v1" >}} and
{{< api-reference page="cluster-resources/lease-candidate-v1alpha1" >}} APIs to elect a leader
as needed.
```

This is a problem though as we don't yet have even a placeholder API reference page for LeaseCandidate.
I suggest adding one and making it clear in this PR that the page is a placeholder to be overwritten when we generate a new API reference.

See https://kubernetes.io/docs/reference/kubernetes-api/cluster-resources/

Reply:
added